theano/gpuarray/opt.py

```diff
@@ -774,6 +773,13 @@ def local_gpua_shape(op, context_name, inputs, outputs):
     return [as_gpuarray_variable(inputs[0], context_name).shape]
 
 
+@register_opt2([tensor.compile.ops.Shape], 'fast_compile')
+def local_gpua_shape_graph(op, context_name, inputs, outputs):
+    # op_lifter will call this opt too frequently as the output is
+    # always on the CPU.
```

Comment:
Remove that comment, it only applied to the `if...: return` lines that are not there.